### PR TITLE
Show delegates acc to the delegate chain id

### PR DESCRIPTION
**Pull request by 0xNilesh**
Backend changes are only compatible for dev env as of now.

### CODE REVIEW

**Review by corlard3y — DISMISSED** [review]

File: src/components/DelegateInfo.js [hunk 1]
```diff
@@ -2,9 +2,10 @@ import React, { useEffect, useState } from "react";
 import styled from "styled-components";
 import { useDeviceWidthCheck } from "hooks";
 import { RiFileCopyFill, RiFileCopyLine } from "react-icons/ri";
-import { Item } from "primaries/SharedStyling";
+import { Image, Item, Span } from "primaries/SharedStyling";
```
Comment (mishramonalisha76) [inline src/components/DelegateInfo.js:5]: would be better to put comments separating the imports, like other files

File: src/components/ShowDelegates.jsx [hunk 2]
```diff
@@ -137,7 +205,7 @@ const ShowDelegates = () => {
   )
 }
 
-const RemoveButton = ({ delegateAddress, removeDelegateModalOpen,showRemoveDelegateModal }) => {
+const InfoButton = ({ clickHandler, hoverText }) => {
```
Comment (mishramonalisha76) [inline src/components/ShowDelegates.jsx:208]: Types for props can be added

File: src/helpers/CaipHelper.ts [hunk 3]
```diff
@@ -30,6 +30,21 @@ export const getCAIP = (chainId: number) => {
   }
 }
 
+export const getCaipToObj = (caip: string) => {
```
Comment (mishramonalisha76) [inline src/helpers/CaipHelper.ts:33]: give type for returned varibale for the function

File: src/helpers/UtilityHelper.ts [hunk 4]
```diff
@@ -9,6 +9,16 @@ const UtilityHelper = {
       return true;
     }
   },
+  isPolygon: (chainId: number) => {
```
Comment (mishramonalisha76) [inline src/helpers/UtilityHelper.ts:12]: give type for returned varibale for the function

File: src/helpers/UtilityHelper.ts [hunk 5]
```diff
@@ -9,6 +9,16 @@ const UtilityHelper = {
...
+    if (chainId === 137 || chainId === 80001) {
+      return true;
+    }
+  },
+  isEthereum: (chainId: number) => {
```
Comment (mishramonalisha76) [inline src/helpers/UtilityHelper.ts:17]: give type for returned varibale for the function

File: src/components/DelegateInfo.js (hunk 1 above)
Comment (mishramonalisha76) [inline src/components/DelegateInfo.js:5]: We can sperate out imports groups by comments like in other files

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]

**Comment by 0xNilesh** [comment]
https://github.com/ethereum-push-notification-service/push-storage-node/issues/580

**Pull request closed without merging**